### ISSUE TITLE
Runtime dispatch with `ft_fix_before`

### ISSUE THREAD

**Issue by roflmaostc** [bionanoimaging/FourierTools.jl#29]
Hi,

@RainerHeintzmann there is a couple of runtime dispatches because of type-instability.

See this: https://github.com/bionanoimaging/FourierTools.jl/blob/000735e46a7deb17a71c69ae701508a59dd85108/src/fourier_resizing.jl#L147

I think you probably should always return the `FourierJoin` type, otherwise it can't handle it.

This causes actually some issues with slow packaging loading times.

**Comment by roflmaostc** [comment]
This affects also `rft_pad`, `rft_fix_after`.

**Comment by roflmaostc** [comment]
I used this tool: https://aviatesk.github.io/JET.jl/stable/tutorial/

```
@report_opt ignored_modules=(AnyFrameModule(Base),) resample(x, (2,2))
```

**Comment by RainerHeintzmann** [comment]
Ok. Will see if I can fix this.

**Comment by roflmaostc** [comment]
fixed by #30
